tests/test_e2e_13_mef_eline.py

```diff
@@ -0,0 +1,1117 @@
+        self.net.start_controller(clean_config=True, enable_all=True)
+        self.net.wait_switches_connect()
+        time.sleep(10)
+
+    @classmethod
+    def setup_class(cls):
+        cls.net = NetworkTest(CONTROLLER)
+        cls.net.start()
+        cls.net.restart_kytos_clean()
+        cls.net.wait_switches_connect()
+        time.sleep(5)
+
+    @classmethod
+    def teardown_class(cls):
+        cls.net.stop()
+
+    def create_evc(self, vlan_id, store=False):
+        payload = {
+            "name": "Vlan_%s" % vlan_id,
+            "enabled": True,
+            "dynamic_backup_path": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:01:1",
+                "tag": {"tag_type": 1, "value": vlan_id}
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:02:1",
+                "tag": {"tag_type": 1, "value": vlan_id}
+            }
+        }
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
+        data = response.json()
+        if store:
+            self.evcs[vlan_id] = data['circuit_id']
+        return data['circuit_id']
+
+    def test_005_patch_unknown_circuit(self):
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        primary_path = data['primary_path']
+
+        payload = {
+            "primary_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:3"}}
+            ]
+        }
+
+        # It sets a new circuit's primary_path
+        response = requests.patch(api_url + evc1 + "A", data=json.dumps(payload),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 404
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['primary_path'] == primary_path
+
+    def test_010_patch_an_empty_uni_a(self):
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        payload = {
+            "uni_a": {}
+        }
+
+        # It tries to setting up a new uni_a
+        response = requests.patch(api_url + evc1, data=json.dumps(payload),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 400
+
+    def test_015_patch_an_inconsistent_uni_a(self):
+        """ No existing switch """
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        payload = {
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:04:1"
```

Comment:
```suggestion
                "interface_id": "00:00:00:00:00:00:00:99:1"
```

Just a suggestion to use a completely different dpid, so the reader don't gets confused (the dpid 04 is valid for another topology)

Reply:
Resolved with the commit [fc88b25](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/fc88b251332d6019379b6fbf1582b5a6381b812f)